### DISCUSSION TITLE
Colorize dead links inside advisories

### DISCUSSION THREAD

**Discussion by jasnow** [cloudsecurityalliance/gsd-tools#179] [Ideas]
Colorize (probably red) dead links inside advisories

**Comment by kurtseifried** [comment]
Where do we keep meta data on the health of a link? Also what states are there? Off the top of my head:

1. site is completely gone (e.g. dns is gone), we can mark as dead easy
2. site DNS was registered to someone else (e.g. I grabbed a few domains that were linked in data), we can't easily mark as dead without content analysis
3. site is still around but content is completely gone, but no 404, e.g. links to a marketing page, we can't easily mark as dead without content analysis
4. site is still around and content 404's, ok, we can mark as dead easy
5. site is still around and content is "correct", we can't easily mark as correct without content analysis

what we need to do is archivebox (or whatever sstem we end up with) all the content in any event because:

just because a URL works, doesn't mean you and I get the same content. If we're going to refer to something, we need to have a copy of it, like Wikipedia does.

**Reply by joshbuker** [reply]
RE: Mirroring references, see also: #139
